packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enhancement
+
+-   Migrate from `nodegit` to `simple-git`.
```

Comment:
Should we make this a breaking change? I think it could break things for folks who don't have a `git` binary (though that would be rare)

Reply:
Maybe. You're right that it is kind of a breaking change :thinking: (OTOH, who will use `wp-env` if they don't have `git` installed? :sweat_smile: )

But yeah, let's make it breaking.